StreamChatSwiftUITests/Infrastructure/TestTools/ChatClient_Mock.swift

```diff
@@ -4,23 +4,25 @@
 
 import Foundation
 @testable import StreamChat
+@testable import StreamChatTestTools
 
 public extension ChatClient {
     /// Create a new instance of mock `ChatClient`
     static func mock(
         isLocalStorageEnabled: Bool = false,
         customCDNClient: CDNClient? = nil
-    ) -> ChatClient {
+    ) -> ChatClient_Mock {
         var config = ChatClientConfig(apiKey: .init("--== Mock ChatClient ==--"))
         config.customCDNClient = customCDNClient
         config.isLocalStorageEnabled = isLocalStorageEnabled
         config.isClientInActiveMode = false
         config.maxAttachmentCountPerMessage = 10
 
-        return .init(
+        return ChatClient_Mock(
```

Comment:
Switched to `StreamChatTestTools` provided `ChatClient_Mock` class since it enables mocking app settings.